addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -171,7 +171,7 @@ def _get_operation_line(self, bom, qty, level):
         total = 0.0
         for operation in bom.operation_ids:
             operation_cycle = float_round(qty / operation.workcenter_id.capacity, precision_rounding=1, rounding_method='UP')
-            duration_expected = operation_cycle * operation.time_cycle + operation.workcenter_id.time_stop + operation.workcenter_id.time_start
+            duration_expected = operation_cycle * operation.time_cycle + ((operation.workcenter_id.time_stop + operation.workcenter_id.time_start) * operation_cycle)
```

Comment:
```suggestion
            duration_expected = operation_cycle * (operation.time_cycle + (operation.workcenter_id.time_stop + operation.workcenter_id.time_start))
```
shorter if you factorize